test/e2e/events_test.go

```diff
@@ -199,4 +199,16 @@ var _ = Describe("Podman events", func() {
 
 		wg.Wait()
 	})
+
+	It("podman events pod creation", func() {
+		SkipIfNotFedora()
+		create := podmanTest.Podman([]string{"pod", "create", "--infra=false", "--name", "foobarpod"})
+		create.WaitWithDefaultTimeout()
+		Expect(create).Should(Exit(0))
+		result := podmanTest.Podman([]string{"events", "--stream=false", "--filter", "type=pod"})
+		result.WaitWithDefaultTimeout()
+		Expect(result).Should(Exit(0))
+		fmt.Println(result.OutputToStringArray())
+		Expect(len(result.OutputToStringArray()) == 1)
```

Comment:
The test is not functional. Please remove the Println() and to check the len you have to use `Expect(result.OutputToStringArray()).To(HaveLen(1))`.
I suggest to also check if the correct pod ID and event type is in the output.

Reply:
`OutPutToStringArray` does not have  `.To` function. the other events test do the same. I can Change to `(len(result.OutputToStringArray()).To(Equal(1)))`

Reply:
fixed